docker/README.md

```diff
@@ -0,0 +1,7 @@
+# Convenient docker images
+
+In this folder we provide Dockerfiles to build docker images with PyTorch, Ignite and other convenient libraries:
+
+- [basic](basic/Dockerfile): latest stable PyTorch, latest stable Ignite, OpenCV, Albumentations, Nvidia/Apex, etc
```

Comment:
Now, the latest version of PyTorch is 1.6.0 and it contains torch native amp. So, no need to install `Nvidia/Apex`. 

Let's provide several docker images (and also rename `basic` to `vision-basic` (if any suggestions to rename it differently, let's see))
- [vision-basic](vision-basic/Dockerfile): latest stable PyTorch, latest stable Ignite, OpenCV, Albumentations, etc
- [vision-basic-apex](vision-basic/Dockerfile.apex): latest stable PyTorch, latest stable Ignite, OpenCV, Albumentations and Nvidia/Apex etc

Those images will have the names:
```
docker pull pytorchignite/vision-basic:latest
docker pull pytorchignite/vision-basic-apex:latest
```
If any other suggestion to regroup things, let's discuss :)
What do you think ?

Reply:
Thanks for the review!
The term `basic` is maybe not mandatory, images name could be just ( but depends if there will other possible prefix ? ) : 
`pytorchignite/vision:latest` and  `pytorchignite/apex-vision:latest`

Reply:
I agree, we can omit "basic". For apex, do you know how to build docker image in multi-steps to reduce image size ? Something like that : 
- start from pytorch devel image
- build apex
- somehow get the artifact
- start from pytorch runtime image
- copy the artifact 
- install artifact

https://docs.docker.com/develop/develop-images/multistage-build/

The last two points I do no know how to perform, that's why I ask...

Reply:
I haven't built an image in multi-steps yet, though I can give it a try. For apex artifact would you recommend conda-pack in order to create a  relocatable environment in the runtime image or have you another idea on how to do that ?

Reply:
Honestly, I don't know. Maybe we have more control by rebuilding ourselves...